kinto/__main__.py

```diff
@@ -23,7 +23,8 @@ def main(args=None):
 
     parser = argparse.ArgumentParser(description="Kinto Command-Line "
                                                  "Interface")
-    commands = ('init', 'start', 'migrate', 'delete-collection', 'version')
+    commands = ('init', 'start', 'migrate', 'delete-collection', 'version',
+                'rebuild-quotas')
```

Comment:
I can add a check to the script to see if it's in `includes`, how does that sound?